config/hsc/multiBandDriver.py

```diff
@@ -2,7 +2,7 @@
 import os.path
 
 
-for sub in ("mergeCoaddDetections", "measureCoaddSources", "mergeCoaddMeasurements", "forcedPhotCoadd"):
+for sub in ("mergeCoaddDetections", "measureCoaddSources", "mergeCoaddMeasurements", "forcedPhotCoadd", "detectCoaddSources"):
```

Comment:
I don’t see a `detectCoaddSources.py` in `config/hsc`, but I suppose it doesn’t hurt to add it here given the if clause.